### PR TITLE
Removed duplicate files, and updated the select_deploy.sh script.

### PR DESCRIPTION
**Pull request by corpulent**
Removed duplicate files, and updated the select_deploy.sh script.

### CODE REVIEW

File: deploy-aws/Makefile [hunk 1]
```diff
@@ -12,6 +12,10 @@ deploy_staging : build_prod
 	eb use ids-com-staging && \
 	. scripts/eb_deploy.sh
 
+deploy_staging_a : build_prod
+	eb use ids-com-staging-a && \
+	. scripts/eb_deploy.sh
+
```
Comment (clepore) [inline deploy-aws/Makefile]: I am confused by having 2 staging's. Which one is the one we should be deploying with?

Comment (corpulent) [reply deploy-aws/Makefile]: @clepore I can decommission the old staging and have the new staging be the official staging.  I was keeping the old one as backup.  Keep this PR open,  I want to monitor the new staging some more,  it seems to be complaining also.

Comment (clepore) [reply deploy-aws/Makefile]: Oh okay

**Comment by nickwynja** [comment]
Approval pending further testing of staging.

**Pull request merged**